qa/tasks/rgw.py

```diff
@@ -134,6 +114,10 @@ def ship_apache_configs(ctx, config, role_endpoints, on_client = None,
     for client in clients_to_create_as:
         if client == except_client:
             continue
+    	cluster_name, daemon_type, client_id = teuthology.split_role(client)
+    	# in case client doesn't have cluster_name attached to it
+    	client_with_id = daemon_type + '.' + client_id
```

Comment:
I think the comment is wrong , you need if for when you provide the cluster separately